### PR TITLE
fix(plugin-meetings): setting remote video quality not working issue

### PR DESCRIPTION
**Pull request by arun3528**
[ ⛔️ ----- REMOVE THIS SECTION  ----- ⛔️

Hey there,\
Thank you for taking the time to improve our code! 🙂\
Please let us know why this change is necessary. \
This ensures our reviewers understand the impact of your change.

⛔️ ----- SECTION END  ----- ⛔️]

Fixes #[INSERT LINK TO ISSUE NUMBER]

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js [hunk 1]
```diff
@@ -70,16 +70,19 @@ const insertBandwidthLimit = (sdpLines, index) => {
...
  */
-const setMaxFs = (sdp, level = QUALITY_LEVELS.HIGH) => {
-  if (!MAX_FRAMESIZES[level]) {
-    throw new ParameterError(`setMaxFs: unable to set max framesize, value for level "${level}" is not defined`);
+const setRemoteVideoConstraints = (sdp, level = QUALITY_LEVELS.HIGH) => {
+  const maxFs = REMOTE_VIDEO_CONSTRAINTS.MAX_FS[level];
+  const maxMbps = REMOTE_VIDEO_CONSTRAINTS.MAX_BITRATE[level];
+
+  if (!maxFs || !maxMbps) {
+    throw new ParameterError(`setRemoteVideoConstraints: unable to set max framesize, value for level "${level}" is not defined`);
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js:78]: this message needs to be updated to include max-mbps and would be useful if it said which one of the 2 is missing

File: packages/node_modules/@webex/plugin-meetings/src/config.js [hunk 2]
```diff
@@ -61,10 +61,10 @@ export default {
       verboseEvents: true
     },
     resolution: {
-      maxWidth: 1280,
-      maxHeight: 720,
-      idealWidth: 1280,
-      idealHeight: 720
+      maxWidth: 1920,
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/src/config.js:64]: are we sure it's fine to suddenly change this default from 720 to 1080 for everybody?

File: packages/node_modules/@webex/plugin-meetings/src/constants.js [hunk 3]
```diff
@@ -1192,16 +1194,22 @@ export const VIDEO_RESOLUTIONS = {
...
-  [QUALITY_LEVELS.LOW]: 1620,
-  [QUALITY_LEVELS.MEDIUM]: 3600,
-  [QUALITY_LEVELS.HIGH]: 8192
+export const REMOTE_VIDEO_CONSTRAINTS = {
+  MAX_FS: {
+    [QUALITY_LEVELS.LOW]: 1620,
+    [QUALITY_LEVELS.MEDIUM]: 3600,
+    [QUALITY_LEVELS.HIGH]: 8192
+  },
+  MAX_BITRATE: {
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/src/constants.js:1206]: the name is wrong - this is not bitrate, "max-mbps" is the decoder's maximum processing speed in macroblocks per second

**Comment by jonniespratley** [comment]
> the title of this PR doesn't seem right... it won't look good in the changelog

Agreed

**Comment by sreenara** [comment]
Closing this old PR as it's taking up a slot in Amplify

**Pull request closed without merging**